Emby.Server.Implementations/TextEncoding/NLangDetect/Extensions/UnicodeBlock.cs

```diff
@@ -1,131 +1,131 @@
 ﻿namespace NLangDetect.Core.Extensions
 {
-  public enum UnicodeBlock
-  {
-    BasicLatin,
-    Latin1Supplement,
-    LatinExtendedA,
-    LatinExtendedB,
-    IpaExtensions,
-    SpacingModifierLetters,
-    CombiningDiacriticalMarks,
-    Greek,
-    Cyrillic,
-    CyrillicSupplementary,
-    Armenian,
-    Hebrew,
-    Arabic,
-    Syriac,
-    Thaana,
-    Devanagari,
-    Bengali,
-    Gurmukhi,
-    Gujarati,
-    Oriya,
-    Tamil,
-    Telugu,
-    Kannada,
-    Malayalam,
-    Sinhala,
-    Thai,
-    Lao,
-    Tibetan,
-    Myanmar,
-    Georgian,
-    HangulJamo,
-    Ethiopic,
-    Cherokee,
-    UnifiedCanadianAboriginalSyllabics,
-    Ogham,
-    Runic,
-    Tagalog,
-    Hanunoo,
-    Buhid,
-    Tagbanwa,
-    Khmer,
-    Mongolian,
-    Limbu,
-    TaiLe,
-    KhmerSymbols,
-    PhoneticExtensions,
-    LatinExtendedAdditional,
-    GreekExtended,
-    GeneralPunctuation,
-    SuperscriptsAndSubscripts,
-    CurrencySymbols,
-    CombiningMarksForSymbols,
-    LetterlikeSymbols,
-    NumberForms,
-    Arrows,
-    MathematicalOperators,
-    MiscellaneousTechnical,
-    ControlPictures,
-    OpticalCharacterRecognition,
-    EnclosedAlphanumerics,
-    BoxDrawing,
-    BlockElements,
-    GeometricShapes,
-    MiscellaneousSymbols,
-    Dingbats,
-    MiscellaneousMathematicalSymbolsA,
-    SupplementalArrowsA,
-    BraillePatterns,
-    SupplementalArrowsB,
-    MiscellaneousMathematicalSymbolsB,
-    SupplementalMathematicalOperators,
-    MiscellaneousSymbolsAndArrows,
-    CjkRadicalsSupplement,
-    KangxiRadicals,
-    IdeographicDescriptionCharacters,
-    CjkSymbolsAndPunctuation,
-    Hiragana,
-    Katakana,
-    Bopomofo,
-    HangulCompatibilityJamo,
-    Kanbun,
-    BopomofoExtended,
-    KatakanaPhoneticExtensions,
-    EnclosedCjkLettersAndMonths,
-    CjkCompatibility,
-    CjkUnifiedIdeographsExtensionA,
-    YijingHexagramSymbols,
-    CjkUnifiedIdeographs,
-    YiSyllables,
-    YiRadicals,
-    HangulSyllables,
-    HighSurrogates,
-    HighPrivateUseSurrogates,
-    LowSurrogates,
-    PrivateUseArea,
-    CjkCompatibilityIdeographs,
-    AlphabeticPresentationForms,
-    ArabicPresentationFormsA,
-    VariationSelectors,
-    CombiningHalfMarks,
-    CjkCompatibilityForms,
-    SmallFormVariants,
-    ArabicPresentationFormsB,
-    HalfwidthAndFullwidthForms,
-    Specials,
-    LinearBSyllabary,
-    LinearBIdeograms,
-    AegeanNumbers,
-    OldItalic,
-    Gothic,
-    Ugaritic,
-    Deseret,
-    Shavian,
-    Osmanya,
-    CypriotSyllabary,
-    ByzantineMusicalSymbols,
-    MusicalSymbols,
-    TaiXuanJingSymbols,
-    MathematicalAlphanumericSymbols,
-    CjkUnifiedIdeographsExtensionB,
-    CjkCompatibilityIdeographsSupplement,
-    Tags,
-    VariationSelectorsSupplement,
-    SupplementaryPrivateUseAreaA,
-    SupplementaryPrivateUseAreaB,
-  }
+    public enum UnicodeBlock
+    {
+        BasicLatin,
```

Comment:
I'd probably put more than one of these on each line to make it more compact, but save for another time :)